packages/manager/apps/ips/src/data/hooks/catalog/catalog.utils.ts

```diff
@@ -1,5 +1,16 @@
+import {
+  isRegionInEu,
+  isRegionInUs,
+} from '@/components/RegionSelector/region-selector.utils';
 import { CatalogIpPlan } from '../../api/catalog';
 
+export const getContinentKeyFromRegion = (region: string) => {
+  if (isRegionInEu(region)) {
+    return 'EU';
+  }
+  return isRegionInUs(region) ? 'US' : 'CA';
+};
+
```

Comment:
For better readability can we use switch ?
```suggestion
export const getContinentKeyFromRegion = (region: string) => {
   switch (region) {
    case isRegionInEu(region):
      return 'EU';
    case isRegionInUs(region):
      return 'US';
    default:
      return 'CA';
  }
};

```

Reply:
Well the `switch(region)` is not usefull since we have to call a function in each case
It would do the same with `switch(true)`

I can just use a second `if` instead of a ternary if you find it clearer to read
```
if (isRegionInEu(region))
  return 'EU';
if (isRegionInUs(region))
  return 'US';
return 'CA';
```